src/service/supplier.service.ts

```diff
@@ -0,0 +1,26 @@
+import { ISupplier, IverifySupplier } from "@/types/supplier";
+import { BaseService } from "./base.service";
+
+const getSupplierList = async (data: ISupplier) => {
+  return await BaseService.fetchData({
+    url: `/supplier`,
+    method: "get",
+    params: data,
+  });
+};
+
+const verifySupplier = async (
+  accessToken: string,
+  Supplier: IverifySupplier
+) => {
+  return await BaseService.fetchData({
+    headers: { Authorization: `Bearer ${accessToken}` },
```

Comment:
This is necessary?

Remember the code in `./base.service.ts`. We made a `request.interceptor ` to mediate the signature of Bearer token in request.